samples/buildrun/buildrun_buildpacks-v3-heroku_cr.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: build.dev/v1alpha1
+kind: BuildRun
+metadata:
+  name: buildpack-nodejs-buildrun-heroku
+spec:
+  buildRef:
+    name: buildpack-nodejs-build-heroku
+  serviceAccount:
+    name: pipeline
+    generate: true
```

Comment:
It does not make sense to specify both a `name` as well as `generate: true`. The code in https://github.com/redhat-developer/build/blob/bc21108d3f6eb22bcfd2675fd780882682e7738d/pkg/controller/buildrun/buildrun_controller.go#L223-L271 will check for the `generate` flag first and if true not check the `name` field at all.

Reply:
if this is the case, this is a bug already in master, lets fix this with another PR in all files!

Reply:
Fine with me.